polkadot/xcm/pallet-xcm-benchmarks/src/generic/mod.rs

```diff
@@ -100,6 +106,13 @@ pub mod pallet {
 				crate_version: <frame_system::Pallet<Self> as frame_support::traits::PalletInfoAccess>::crate_version(),
 			}
 		}
+
+		/// The worst case buy execution weight limit and
+		/// asset to trigger the Trader::buy_execution in the XCM executor
+		/// By default returns ((AssetId(Here.into()), 100_000_000u128), Unlimited)
+		fn worst_case_buy_execution() -> Result<(Asset, WeightLimit), BenchmarkError> {
+			Ok(((AssetId(Junctions::Here.into()), 100_000_000u128).into(), WeightLimit::Unlimited))
```

Comment:
I don't think we need a default for this. More often than not the worst case is `Limited` and we have many examples that don't buy fees with `Here`

Reply:
What should I do in this PR for all the polkadot runtimes? Use the same values they had before? Because I dont think I know wll the polkadot/cumulus  runtimes as well to know what is the worst usecase for each of them..